nise/report.py

```diff
@@ -414,7 +414,8 @@ def aws_create_report(options):  # noqa: C901
         monthly_files = []
         fake = Faker()
         num_gens = len(generators)
-        LOG.info(f"Producing data for {num_gens} generators for {month.get('name')}.")
+        ten_percent = int(num_gens * 0.1) if num_gens > 50 else 5
```

Comment:
instead of 50, should this be a function of the number of CPUs the system has?  e.g. `2 * multiprocessing.cpu_count()`

Reply:
This 50 is just a made up number. The `ten_percent` is used strictly for logging progress:
```
2020-06-01 11:00:40,390 : nise.report.418 : INFO : Producing data for 1750 generators for 2019-06.
2020-06-01 11:00:54,540 : nise.report.455 : INFO : Done with 175 of 1750 generators. <<<<<<
2020-06-01 11:01:12,871 : nise.report.455 : INFO : Done with 350 of 1750 generators.
```

The ten_percent is used to calculate/log how many generators are done.